src/index.coffee

```diff
@@ -1,16 +1,16 @@
-isPathAbsolute = require 'path-is-absolute'

```

Comment:
Remove the dependency from package.json

Reply:
fixed by https://github.com/Oxalin/node-auto-launch/commit/e46e9092cf84d0c06570bbcc621bb96358c67290